packages/ambassador-council/hardhat.config.js

```diff
@@ -24,11 +24,11 @@ const config = {
       url: 'http://localhost:8545',
     },
     ['optimistic-kovan']: {
-      url: 'https://kovan.optimism.io',
+      url: process.env.NETWORK_ENDPOINT || 'https://kovan.optimism.io',
```

Comment:
Updated all hardhat files to allow to change the url, in the case that the user want's to connect to a fork, but interact with an existing deployment.